packages/usdk/packages/upstreet-agent/packages/react-agents/components/util/default-components.tsx

```diff
@@ -120,21 +120,24 @@ const CharactersPrompt = () => {
   const agent = useAgent();
   const name = useName();
   const bio = usePersonality();
+  const [agents, setAgents] = useState([]);
   if (conversation) {
-    const agents = conversation.getAgents();
     const currentAgentSpec = {
       id: agent.id,
       name,
       bio,
     };
-    const agentSpecs = agents.map((agent) =>  {
-      const agentSpec = agent.getPlayerSpec() as any;
-      return {
-        name: agentSpec?.name,
-        id: agent.playerId,
-        bio: agentSpec?.bio,
-      };
-    });
+  
+    const agentSpecs = agents
+      .filter(mapAgent => mapAgent.playerId !== agent.id)
```

Comment:
We don't want the current agent in this list?

Reply:
The current agent is being added via the currentAgentSpec + formatAgent inside the Prompt component, agentSpecs mapped out for "Other Characters" hence need to exclude the current agent